lib/bookingsync/api/client.rb

```diff
@@ -314,7 +314,11 @@ def middleware
     end
 
     def faraday_options
```

Comment:
Maybe better to allow top override them? i.e. replace this part of initializer:
`@conn = Faraday.new(faraday_options)`
with:
`@conn = Faraday.new(faraday_options.merge(options.fetch(:faraday_options, {})))` (but in more readable way )